src/unix/notbsd/linux/other/mod.rs

```diff
@@ -1,5 +1,18 @@
 pub type __priority_which_t = ::c_uint;
 
+s_no_extra_traits! {
+    pub union XXX_not_sure_what_to_call_this {
```

Comment:
How is this called in C ? Or is it an anonymous `union` ?

Reply:
Anonymous :-(

Reply:
Then maybe just call it `sigfault_t_anonymous_union`. It's a private field, so I don't think we have to make this type public, right?

Reply:
The `s!` macro only works with `pub` types, so I just went that direction for simplicity.

Reply:
I think it would be ok to implement these outside of `s!`.